components/filter/filter-dimension-set-date-time-range-value.js

```diff
@@ -39,14 +41,20 @@ class FilterDimensionSetDateTimeRangeValue extends LocalizeCoreElement(LitElemen
 			 * Defaults to "Custom Date Range" (localized). Can be overridden if desired.
 			 * @type {string}
 			 */
-			text: { type: String, reflect: true }
+			text: { type: String, reflect: true },
+			/**
+			 * Date/time range input type
+			 * @type {'date'|'date-time'}
+			 */
+			type: { type: String }
```

Comment:
Another option would be to have a bool like `hide-time`. The advantage of `type` is in case in the future we need to do some sort of time-only range or some other state (though there are currently no plans for those)

Reply:
Of the two, I prefer the `type` approach for the reason you mentioned.

Another alternative we can contemplate is busting it into two different components, which might be overkill if the API is essentially the same and there isn't much in the way of conditional logic.

Reply:
Agreed - at this point (nor going forward) I don't see it as necessary to split them out.